README.md

```diff
@@ -90,6 +90,21 @@ The supported algorithms can be extended at runtime by using [cose.RegisterAlgor
 
 [API docs](https://pkg.go.dev/github.com/veraison/go-cose)
 
+### Integer Ranges
+
+CBOR supports integers in the range:
+
+```text
+[-2<sup>64</sup>, -1] ∪ [0, 2<sup>64</sup> - 1]
```

Comment:
@qmuntal `<sup>` and `</sup>` are not rendered in text code block.

We can try [Unicode superscripts](https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts#Superscripts_and_subscripts_block) like `[-2⁶⁴, -1] ∪ [0, 2⁶⁴ - 1]`.

Reply:
@shizhMSFT , can we log an issue to capture this?

Reply:
I'll just submit a PR fixing this rendering.

Reply:
> We can try [Unicode superscripts](https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts#Superscripts_and_subscripts_block) like [-2⁶⁴, -1] ∪ [0, 2⁶⁴ - 1].

Unicode superscripts are not rendered in some mobile browsers, at least not in mine, so I'll probably just move the range outside the code block.